app/models/manageiq/providers/openstack/infra_manager/host.rb

```diff
@@ -455,4 +455,31 @@ def refresh_network_interfaces(ssu)
   def self.display_name(number = 1)
     n_('Host (OpenStack)', 'Hosts (OpenStack)', number)
   end
+
+  def self.post_refresh_ems(ems_id, update_start_time)
+    ems = ExtManagementSystem.find(ems_id)
+    hosts = ems.hosts.where("created_on >= ?", update_start_time)
+    hosts.find_each(&:post_create_actions_queue) unless hosts.empty?
```

Comment:
`.where` will return an empty relation so no need to worry about if `.empty?` since `[].find_each(&:....)` will just be a noop